script.js

```diff
@@ -0,0 +1,21 @@
+const element = document.getElementById('hamNenu');
```

Comment:
- [ ] Let's try to be more specific about naming elements in our code. This could be named `burguerIcon` or `hamIcon`. This is helpful when you need to come back to your code and find out what a piece of code does. If you're working with a lot of variables in the same scope it might help to avoid bugs by not naming things the same. [You can read more about JavaScript best practices here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)